roles/host-provision/tasks/rhel-config-tasks.yml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: config-tasks | Validate internet connectivity via control node
+  uri:
+    url: https://www.google.com
+    status_code: 200
+  tags: host-provision
+
+- name: config-tasks | Get current registration status
+  become: true
+  shell: |
+    /sbin/subscription-manager list |grep Status: |awk '{print $2}'
```

Comment:
This command returns blank is there are no installed products;  I'm not quite sure what the point of printing the current status is, but if it's necessary, would it be more useful to print out the full subscription-manager list output?

Reply:
Removing the print block.

Additional info below:
When the system is unregistered:
```
[root@bmshost pki]# subscription-manager list
+-------------------------------------------+
    Installed Product Status
+-------------------------------------------+
Product Name:   Red Hat Enterprise Linux Server
Product ID:     69
Version:        7.8
Arch:           x86_64
Status:         Unknown
Status Details: 
Starts:         
Ends:           
```

When the system is subscribed:
```
[root@bmshost pki]# subscription-manager list
+-------------------------------------------+
    Installed Product Status
+-------------------------------------------+
Product Name:   Red Hat Enterprise Linux Server
Product ID:     69
Version:        7.8
Arch:           x86_64
Status:         Subscribed
Status Details: 
Starts:         04/20/2021
Ends:           08/20/2022
```

Reply:
Done and also printing the output of entire `subscription-manager` for clarity, as originally suggested.